Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -1021,6 +1068,26 @@ public void SetIMEEnabled(bool enabled)
         ///
         /// See <see cref="Keyboard.SetIMEEnabled"/> for turning IME on/off
         /// </remarks>
+        /// <param name="position">
+        ///     A Vector2 of the IME cursor position to set.
+        /// </param>
+        /// <example>
+        /// <code>
+        /// public class KeyboardUtils : MonoBehaviour
+        /// {
+        ///     private Vector2 cursorPosition;
+        ///
+        ///     void Update ()
+        ///     {
+        ///         // Set the IME cursor position to the mouse position
+        ///         var x = Input.GetAxis("Mouse X");
```

Comment:
Why is this using Input Manager? Wouldn't it make sense to use input system instead?

Reply:
Ups, old habit - exactly me should know better now after working on the deprecation for so long :D - fixed in [95c3c3e](https://github.com/Unity-Technologies/InputSystem/pull/2069/commits/95c3c3ed1d50602d58b99c640ac82ca281e4f825)